dns/pfSense-pkg-bind/files/usr/local/pkg/bind.inc

```diff
@@ -450,7 +450,10 @@ EOD;
 					case 'slave':
 						$bind_conf .= "\t\tmasters { $zoneipslave; };\n";
 						$bind_conf .= "\t\tallow-query { $zoneallowquery; };\n";
-						$bind_conf .= "\t\tallow-transfer { $zoneallowtransfer; };\n";
+						//$bind_conf .= "\t\tallow-transfer { $zoneallowtransfer; };\n";
+							if ( $zonecustom == '' or ($zonecustom != '' and !preg_match('/allow-transfer/', $zonecustom))) {
```

Comment:
You don't need to test if `$zonecustom` is empty or not, just call preg_match is enough here. Please also fix indentation, it's one level ahead previous lines

Reply:
You are right :)